src/riak_kv_ts_svc.erl

```diff
@@ -801,7 +801,7 @@ check_table_and_call(Table, Fun, TsMessage, State) ->
 -spec make_rpberrresp(integer(), string()) -> #rpberrorresp{}.
 make_rpberrresp(Code, Message) ->
     #rpberrorresp{errcode = Code,
-                  errmsg = lists:flatten(Message)}.
+                  errmsg = iolist_to_binary(Message)}.
```

Comment:
Nope. `iolist_to_binary` has digestive system strong enough to crunch that: http://erldocs.com/R16B02/erts/erlang.html?i=1&search=iolist#iolist_to_binary/1.